src/main/java/justbot/command/MarkCommand.java

```diff
@@ -0,0 +1,58 @@
+package justbot.command;
+
+import justbot.exception.JustbotException;
+import justbot.storage.Storage;
+import justbot.task.Task;
+import justbot.task.TaskList;
+import justbot.ui.Ui;
+
+/**
+ * Represents a command to mark a task as done in the Justbot application.
+ * The MarkCommand marks a specific task in the task list as completed.
+ */
+public class MarkCommand extends Command {
+    private int markNumber;
+
+    /**
+     * Constructs a MarkCommand with the specified task number to mark as done.
+     *
+     * @param markNumber The 1-based index of the task to be marked as done.
+     */
+    public MarkCommand(int markNumber) {
```

Comment:
Similar to the comment below, I also believe that taskIndexToMark will be more descriptive! 😄